### PR TITLE
Service-maps-AI&ML

### PR DESCRIPTION
**Pull request by alphadev4**

### CODE REVIEW

File: helpers/google/api.js [hunk 1]
```diff
@@ -181,6 +181,15 @@ var serviceMap = {
             BridgeArnIdentifier: '', BridgeIdTemplate: '', BridgeResourceType: 'b',
             BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'gcp-Storage',
             BridgeCollectionService: 'gcp-buckets', DataIdentifier: 'data',
+        },
+    'AI & ML':
+        {
+            enabled: true, isSingleSource: true, InvAsset: 'models', InvService: 'vertexAI',
+            InvResourceCategory: 'ai&ml', InvResourceType: 'VertexAI models', BridgeServiceName: 'vertexAI',
+            BridgePluginCategoryName: 'AI & ML', BridgeProvider: 'Google', BridgeCall: 'listDatasets',
```
Comment (mehakseedat63) [inline helpers/google/api.js:189]:
```suggestion
            BridgePluginCategoryName: 'AI & ML', BridgeProvider: 'Google', BridgeCall: 'listModels',
```

**Review by mehakseedat63 — CHANGES_REQUESTED** [review]

File: helpers/google/api.js [hunk 2]
```diff
@@ -181,6 +181,15 @@ var serviceMap = {
...
             BridgeCollectionService: 'gcp-buckets', DataIdentifier: 'data',
+        },
+    'AI & ML':
+        {
+            enabled: true, isSingleSource: true, InvAsset: 'models', InvService: 'vertexAI',
+            InvResourceCategory: 'ai&ml', InvResourceType: 'VertexAI models', BridgeServiceName: 'vertexAI',
+            BridgePluginCategoryName: 'AI & ML', BridgeProvider: 'Google', BridgeCall: 'listModels',
+            BridgeArnIdentifier: '', BridgeIdTemplate: '', BridgeResourceType: 'models',
+            BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'AI & ML',
+            BridgeCollectionService: 'vertexai', DataIdentifier: 'data',
```
Comment (mehakseedat63) [inline helpers/google/api.js:192]:
```suggestion
            enabled: true, isSingleSource: true, InvAsset: 'models', InvService: 'vertexAI',
            InvResourceCategory: 'ai&ml', InvResourceType: 'VertexAI models', BridgeServiceName: 'vertexAI',
            BridgePluginCategoryName: 'AI & ML', BridgeProvider: 'Google', BridgeCall: 'listModels',
            BridgeArnIdentifier: '', BridgeIdTemplate: '{name}', BridgeResourceType: 'models',
            BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'AI & ML',
            BridgeCollectionService: 'gcp-vertexai', DataIdentifier: 'data',
```
we also need to add this in supported collection service

**Review by mehakseedat63 — CHANGES_REQUESTED** [review]

File: helpers/google/api.js [hunk 3]
```diff
@@ -181,6 +181,15 @@ var serviceMap = {
...
             BridgeCollectionService: 'gcp-buckets', DataIdentifier: 'data',
+        },
+    'AI & ML':
+        {
+            enabled: true, isSingleSource: true, InvAsset: 'models', InvService: 'vertexAI',
+            InvResourceCategory: 'ai&ml', InvResourceType: 'VertexAI models', BridgeServiceName: 'vertexAI',
+            BridgePluginCategoryName: 'AI & ML', BridgeProvider: 'Google', BridgeCall: 'listModels',
+            BridgeArnIdentifier: '', BridgeIdTemplate: '{name}', BridgeResourceType: 'models',
+            BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'AI & ML',
+            BridgeCollectionService: 'gcp-vertexai', DataIdentifier: 'data',
```
Comment (alphadev4) [inline helpers/google/api.js:192]:
```suggestion
    'AI & ML':
        {
            enabled: true, isSingleSource: true, InvAsset: 'models', InvService: 'vertexAI',
            InvResourceCategory: 'ai&ml', InvResourceType: 'VertexAI models', BridgeServiceName: 'vertexAI',
            BridgePluginCategoryName: 'gcp-AI & ML', BridgeProvider: 'Google', BridgeCall: 'listModels',
            BridgeArnIdentifier: '', BridgeIdTemplate: '{name}', BridgeResourceType: 'models',
            BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'gcp-AI & ML',
            BridgeCollectionService: 'gcp-vertexai', DataIdentifier: 'data',
```

**Review by mehakseedat63 — APPROVED** [review]

File: helpers/azure/api.js [hunk 4]
```diff
@@ -161,6 +161,15 @@ var serviceMap = {
             BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'SQL Databases',
             BridgeCollectionService: 'databases', DataIdentifier: 'data',
         },
+    'AI & ML': 
+        {
+            enabled: true, isSingleSource: true, InvAsset: 'account', InvService: 'openAI',
+            InvResourceCategory: 'ai&ml', InvResourceType: 'OpenAI', BridgeProvider: 'Azure',
```
Comment (alphadev4) [inline helpers/azure/api.js:167]:
```suggestion
            InvResourceCategory: 'ai&ml', InvResourceType: 'OpenAI Accounts', BridgeProvider: 'Azure', 
```

**Review by mehakseedat63 — APPROVED** [review]

**Pull request merged**